modules/migrating-br-ex-bridge-nmstate.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-postinstallation-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrating-br-ex-bridge-nmstate_{context}"]
+= Migrating a configured br-ex bridge to NMState
+
+If you used the `configure-ovs.sh` shell script to set a `br-ex` bridge during cluster installation, you can migrate the `br-ex` bridge to NMState as a postinstallation task. NMState provides a declarative and idempotent way to handle configuring the `br-ex` bridge.
+
+[NOTE]
+====
+The initial steps in the procedure do not show example configurations. For detailed example configurations that would represent objects to create during cluster installation, see the "Creating a manifest object that includes a customized br-ex bridge" link in the _Additional resources_ section.
+====
+
+After you migrate your configured `br-ex` bridge to NMState, you cannot reverse the operation. This means that you cannot migrate back to the shell script version of the `br-ex` bridge.
+
+[IMPORTANT]
+====
+Misconfiguring any files that form part of the migration operation can cause disruptive changes to your cluster. Reverting these changes might not always be possible. 
+====
+
+.Prerequisties 
+
+* You used the `configure-ovs.sh` shell script to set a `br-ex` bridge for your cluster.
+
+.Procedure
+
+. Create an NMState configuration file for your customized `br-ex` bridge network. In a later step, the `MachineConfig` object saves the NMState configuration file in the `/etc/nmstate/openshift` directory path.
+
+. Use the `cat` command to base64-encode the contents of the NMState configuration file:
++
+[source,terminal]
+----
+$ cat <nmstate_configuration>.yaml | base64
+----
++
+--
+where:
+
+`<nmstate_configuration>`:: Replace `<nmstate_configuration>` with the name of your NMState resource YAML file.
```

Comment:
Can you check the rendering here to make sure its what you want?

Also it might be as follows according to our doc guidelines (Not sure if this rule has changed as weve kearned more about callouts):

```suggestion
`<nmstate_configuration>`:: Specifies the name of your NMState resource YAML file.
```
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks